src/iocore/net/test_certlookup.cc

```diff
@@ -234,7 +234,7 @@ main(int argc, const char **argv)
   ink_freelists_dump(stdout);
 
 // On Darwin, fail the tests if we have any memory leaks.
```

Comment:
Maybe change the deferences to Darwin in comments to macOS

Reply:
I thought about it, but it's not actually wrong either (and `uname` returns `Darwin` on macOS). It's really just the preprocessor directives that should be changed. Also, a number of places look for the `Darwin` string as well for various purposes.